pom.xml

```diff
@@ -935,9 +950,9 @@
                             <arg>10000</arg>
                         </compilerArgs>
                         <parameters>true</parameters>
-                        <source>${version.jdk}</source>
-                        <target>${version.jdk}</target>
                         <showWarnings>false</showWarnings>
+                        <source>${version.jdk.target}</source>
+                        <target>${version.jdk.target}</target>
```

Comment:
Is this intended? If so, can we add a comment explaining why source and target are both target? I have a hunch this is correct, yet it still looks super misleading. :eyes:

Reply:
It is intended indeed. `-source` must always be less than or equal to `-target`. `-target` specifies the minimum Java version targeted; that's still 11 as before. Will add a commit.